crates/amaru-consensus/Cargo.toml

```diff
@@ -30,6 +30,7 @@ amaru-kernel.workspace = true
 amaru-ledger.workspace = true
 amaru-ouroboros.workspace = true
 amaru-ouroboros-traits.workspace = true
+acto = { version = "0.7.2", features = ["tokio"] }
```

Comment:
Please declare the dependency version in the top-level Cargo.toml, following the existing pattern 🙏

Reply:
sure, this was temporary while I wasn’t sure whether `acto` is an acceptable dependency